addusers.py

```diff
@@ -285,7 +256,13 @@ def parse_rows(rows):
     return projects, bad_rows
 
 if __name__ == "__main__":
-    openstack = Openstack(admin_user, admin_pwd, admin_tenant, auth_url, nova_version)
+    auth = v3.Password(auth_url=auth_url,
+                       username=admin_user,
+                       user_domain_id = 'default',
+                       password=admin_pwd)
```

Comment:
@xuhang57 I think you are right.  My test environment is still at Liberty and that worked fine without it, but I tried this method of authentication against Mitaka and it restricts what you can do without specifying the project.